api/server.go

```diff
@@ -2,10 +2,13 @@ package api
 
 import (
 	"encoding/json"
+	"github.com/julienschmidt/httprouter"
+	"go.sia.tech/core/consensus"
+	"go.sia.tech/siad/v2/p2p"
+	"go.sia.tech/siad/v2/wallet"
```

Comment:
`goimports` should automatically group these with the other non-stdlib packages at the bottom